corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js

```diff
@@ -47,6 +47,7 @@ hqDefine('app_manager/js/forms/advanced/case_config_ui', function () {
             self.setPropertiesMap(params.propertiesMap);
 
             self.descriptionDict = params.propertyDescriptions;
+            self.deprecatedPropertiesDict = params.deprecatedProperties;
```

Comment:
@zandre-eng So the `deprecatedProperties` is already present in the params, right?

Reply:
Correct, it is being sent from the back-end [here](https://github.com/dimagi/commcare-hq/blob/c61b7de6b413f4262431e8cf7bd17d662b4c7e84/corehq/apps/app_manager/views/forms.py#L794). These params are also getting loaded for normal modules [here](https://github.com/dimagi/commcare-hq/blob/c61b7de6b413f4262431e8cf7bd17d662b4c7e84/corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js#L53).